metadata-ws/src/test/java/uk/ac/ebi/ampt2d/metadata/cucumber/StudySteps.java

```diff
@@ -41,6 +41,13 @@
     @Autowired
     private MockMvc mockMvc;
 
+    @When("I create a study with (.*) for taxonomy and (.*) for accession$")
```

Comment:
This is a very good point since it reduces code duplication. Now fixed in https://github.com/EBIvariation/metadata-ws/pull/53/commits/0934a756cf59798fe498d1d26c16389a1c498009.